### PR TITLE
Feature/revsdl 1289 hmilevel passenger app

### PR DESCRIPTION
**Pull request by pvysh9**
Implement specific rules for HMI Level assigning for apps from passenger's device (see REVSDL-1278) + fix REVSDL-1310

### CODE REVIEW

File: src/components/can_cooperation/src/command/base_command_request.cc [hunk 1]
```diff
@@ -291,9 +291,9 @@ void BaseCommandRequest::Run() {
   LOG4CXX_AUTO_TRACE(logger_);
   if (Validate()) {
     LOG4CXX_INFO(logger_, "Request message validated successfully!");
-    if (CheckPolicy()) {
-      Execute();  // run child's logic
-    }
+      if(CheckPolicy()) {
```
Comment (bonewell) [inline src/components/can_cooperation/src/command/base_command_request.cc]: so much spaces

File: src/components/can_cooperation/src/module_helper.cc [hunk 2]
```diff
@@ -34,12 +34,48 @@
...
+
+  application_manager::MessagePtr message(
+    new application_manager::Message(
+      protocol_handler::MessagePriority::kDefault));
+  message->set_protocol_version(
+    application_manager::ProtocolVersion::kHMI);
+  message->set_correlation_id(msg[json_keys::kId].asInt());
+  Json::FastWriter writer;
+  std::string json_msg = writer.write(msg);
+  printf("MESSAGE \n\t%s\n", json_msg.c_str());
```
Comment (bonewell) [inline src/components/can_cooperation/src/module_helper.cc]: Why didn't you use LOG4CXX?

File: src/components/can_cooperation/test/src/can_module_test.cc [hunk 3]
```diff
@@ -83,8 +83,8 @@ TEST_F(CanModuleTest, ProcessMessageEmptyAppsList) {
   message->set_function_id(MobileFunctionID::ON_INTERIOR_VEHICLE_DATA);
 
   std::vector<application_manager::ApplicationSharedPtr> apps;
-  EXPECT_CALL(*mock_service, GetApplications(module->GetModuleID())
-    ).Times(1).WillOnce(Return(apps));
+  /*EXPECT_CALL(*mock_service, GetApplications(module->GetModuleID())
```
Comment (bonewell) [inline src/components/can_cooperation/test/src/can_module_test.cc]: Can you fix tests?

**Comment by bonewell** [comment]
Reviewed.

**Comment by VladSemenyuk** [comment]
Reviewed

**Pull request merged**